config/locales/en.yml

```diff
@@ -9,3 +9,13 @@ en:
   design: "Revise design"
   dimensions: "Revise dimensions"
   other: "Other"
+
+  activerecord:
+    errors:
+      models:
+        drawing:
+          attributes:
+            plan:
+              unsupported_file_type: "Plan is not a supported file format"
```

Comment:
Should we add text to say which formats are permitted?

Reply:
Updated in https://github.com/unboxed/bops/pull/107/commits/25a590ebb7e53196b51a0fe8668ed735499a4df8